pkg/skaffold/update/update.go

```diff
@@ -43,23 +42,28 @@ func IsUpdateCheckEnabled() bool {
 	return v == "" || strings.ToLower(v) == "true"
 }
 
-// GetLatestVersion uses a VERSION file stored on GCS to determine the latest released version of skaffold
-func GetLatestVersion(ctx context.Context) (semver.Version, error) {
+// VersionCheck uses a VERSION file stored on GCS to determine the latest released version of skaffold
+func VersionCheck() (semver.Version, bool, error) {
 	resp, err := http.Get(latestVersionURL)
 	if err != nil {
-		return semver.Version{}, errors.Wrap(err, "getting latest version info from GCS")
+		return semver.Version{}, false, errors.Wrap(err, "getting latest version info from GCS")
 	}
 	defer resp.Body.Close()
 	if resp.StatusCode != http.StatusOK {
-		return semver.Version{}, errors.Wrapf(err, "http %d, error: %s", resp.StatusCode, resp.Status)
+		return semver.Version{}, false, errors.Wrapf(err, "http %d, error: %s", resp.StatusCode, resp.Status)
 	}
 	versionBytes, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return semver.Version{}, errors.Wrap(err, "reading version file from GCS")
+		return semver.Version{}, false, errors.Wrap(err, "reading version file from GCS")
 	}
-	v, err := version.ParseVersion(string(versionBytes))
+	latest, err := version.ParseVersion(string(versionBytes))
 	if err != nil {
-		return semver.Version{}, errors.Wrap(err, "parsing latest version from GCS")
+		return semver.Version{}, false, errors.Wrap(err, "parsing latest version from GCS")
 	}
-	return v, nil
+	current, err := version.ParseVersion(version.Get().Version)
+	if err != nil {
+		return semver.Version{}, false, errors.Wrap(err, "parsing current semver, skipping update check")
+	}
+	isAhead := latest.GT(current)
```

Comment:
nit: can combine this with the next line to avoid creating a new variable

Reply:
yeah, I also thought about this. Should be changed now.